app/views/sessions/new.erb

```diff
@@ -0,0 +1,11 @@
+<form action="/sessions" method="post">
+  
+  <fieldset>
+    <legend>Log in with password and username or email</legend>
+    <label>username or email</label>
```

Comment:
Good using labels, but if your label element does not wrap around your input element then you need to use the `for` attribute so that screen readers know which input field your form refers to (this helps people with accessibility issues